run_unittests.py

```diff
@@ -3620,6 +3620,35 @@ def test_native_dep_pkgconfig(self):
         self.wipe()
         self.init(testdir, extra_args=['-Dstart_native=true'], override_envvars=env)
 
+    @skipIfNoPkgconfig
+    @unittest.skipIf(is_windows(), 'Help needed with fixing this test on windows')
+    def test_pkg_config_libdir(self):
+        testdir = os.path.join(self.unit_test_dir,
+                               '46 native dep pkgconfig var')
+        with tempfile.NamedTemporaryFile(mode='w', delete=False) as crossfile:
+            crossfile.write(textwrap.dedent(
+                '''[binaries]
+                pkgconfig = r'{0}'
```

Comment:
small nitpicking: you can directly write `pkgconfig = 'pkg-config'` here instead of using a format.